### PR TITLE
Fix settings not reflected on update

### PR DESCRIPTION
**Pull request by bhardwajaditya**
Closes: https://github.com/WideChat/Rocket.Chat/issues/1124

### CODE REVIEW

File: lib/Settings.ts [hunk 1]
```diff
@@ -14,13 +14,6 @@ export const getServerSettingValue = async (read: IRead, id: string) => {
 export const getLivechatAgentConfig = async (read: IRead, sessionId: string, type?: string) => {
 
     try {
-        const persistentAgentConfig = await getPersistentAgentConfigToRoom(read, sessionId);
-        if (persistentAgentConfig) {
-            if (type) {
-                return persistentAgentConfig[type];
-            }
-            return persistentAgentConfig;
-        }
```
Comment (Shailesh351) [inline lib/Settings.ts:17]: @bhardwajaditya This affects connection draining functionality from this PR https://github.com/WideChat/Apps.Dialogflow/pull/120.

The setting should be reflected for new chats/sessions. For the ongoing chats/sessions it should have the same setting when the chat was started

**Comment by Shailesh351** [comment]
Setting not being updated is a generic issue with all app settings. i.e. agent config, log level, any setting of any other app.

**Comment by bhardwajaditya** [comment]
@Shailesh351 I checked with salesforce app and the changes work fine without restarting the server. 

Log level issue seems to be a regression. Logs are not abiding with the setting at all every time all logs are available doesn't depend on the setting.

File: lib/Settings.ts (hunk 1 above)
Comment (bhardwajaditya) [reply lib/Settings.ts:17]: @Shailesh351 if I open the widget and then change the settings, it returns the old setting value only in all the chats for that session.

I will test what happens when I open the widget in incognito. If the settings change work fine there , is that an acceptable situation?

Comment (ear-dev) [reply lib/Settings.ts:17]: @Shailesh351 is correct about 'connection draining'.  Any open rooms should continue to use the same config that started their conversation.  Any new rooms should use the new config. 

Unfortunately, the behavior that I have observed is flaky.  It frequently does not take the new settings until you restart the server, but sometimes it works.......  flaky......

**Comment by Shailesh351** [comment]
Closing in favor of https://github.com/WideChat/Rocket.Chat/pull/1132

**Pull request closed without merging**